src/pages/MovieDetailsPage.vue

```diff
@@ -38,10 +38,9 @@ import DateButton from "../components/DateButton.vue";
 export default {
   name: "MovieDetailsPage",
   metaInfo: {
-    title: () => {
-      return this.movie.title;
-    },
+    title: this.movie.title,
```

Comment:
this probably isn't working too, is it? 🤔  please make sure you test it next time - it should be visible e.g. as browser's tab title:
<img width="886" alt="Screenshot 2022-03-26 at 19 46 11" src="https://user-images.githubusercontent.com/4105687/160253113-1d355b0f-4b35-4282-b1a1-f9861f2eff5d.png">